source/stylesheets/card.scss

```diff
@@ -33,3 +29,52 @@ p.card__prompt {
   font-size: $base__font-size;
   line-height: $base-line-height;
 }
+
+.card__refresh-button {
+  background-color: $blue-fill;
+  border-radius: $base-border-radius;
+  border: $base-border;
+  color: $white;
+  overflow: hidden;
+  padding: .75rem 2rem;
+  position: relative;
+  text-decoration: none;
+  transition: all 0.2s linear 0s;
+
+  &:before {
+    align-items: center;
+    background-color: rgba( #000,0.3);
+    border-radius: 0 50% 50% 0;
+    color: $yellow;
+    content: "\2191";
+    display: inline-flex;
+    font-size: $base__font-size;
+    font-weight: $h1-font-weight;
+    height: 100%;
+    justify-content: center;
+    left: 0px;
+    position: absolute;
+    top: 0;
+    transform-origin: left center;
+    transform: scale(0,1);
+    transition: all 0.2s linear 0s;
+    width: 30px;
+  }
+
+  &:hover {
+    background-color: $dark-purple;
+    color: $white;
+    cursor: pointer;
+    font-weight: bolder;
+    text-indent: 30px;
```

Comment:
This text-indent stuff seems a little smelly. Can flexbox handle the alignment for us?